index.d.ts

```diff
@@ -1,3 +1,23 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, @typescript-eslint/prefer-ts-expect-error
+// @ts-ignore It’s important to preserve this ignore statement. This makes sure
+// it works both with and without node types.
```

Comment:
Is typescript okay with having another comment between the ignore and the import expression?

Reply:
Yep. TypeScript comments apply to the next statement. This is different from `eslint-disable-next-line` comments, which apply to the next line. I guess that’s what you had in mind. :)